test/servico/cst03Test.java

```diff
@@ -53,30 +51,34 @@ public void tearDown() {
     }
     
     @Test
-    public void testeNotificaçãoDeAusência_SemMotivo() throws ParseException
-    {
+    public void testeNotificaçãoDeAusência_SemMotivo() throws ParseException{

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/calebepb/substituicao/file/2738690362/issues/source?bid=3308597&fileBranchId=3353674#l54)